tests/app/models/DevicesModelTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Models\DevicesModel;
+
+class DeicesModelTest extends BaseModelTest
+{
+    public function testConstruct()
```

Comment:
This doesn't seem to fulfill a purpose.  All this does is declare a local variable.  Is there a reason for this?